shared/constants/permissions.js

```diff
@@ -5,6 +5,7 @@ export const CaveatTypes = Object.freeze({
 export const RestrictedMethods = Object.freeze({
   eth_accounts: 'eth_accounts',
   ///: BEGIN:ONLY_INCLUDE_IN(flask)
+  wallet_getLocale: 'wallet_getLocale',
```

Comment:
If it is restricted it needs to be added to https://github.com/MetaMask/metamask-extension/blob/8b9b9af13fabcef4882d4423e35acb4cd3aef4e9/ui/helpers/utils/permission.js